libcxx/test/std/algorithms/alg.nonmodifying/alg.find.last/ranges.find_last_if.pass.cpp

```diff
@@ -183,8 +183,17 @@ struct NonConstComparable {
   friend constexpr bool operator==(NonConstComparable&, const NonConstComparable&) { return true; }
 };
 
+// note: this should really use `std::const_iterator`
```

Comment:
Can you open an issue and change this to a TODO instead please? AIUI we're only not doing that because `const_iterator` isn't around yet.

Reply:
Sure.

Reply:
Oh, I meant using the issue in the todo as `TODO(<#issue-number>): <reason>`.

Reply:
> Oh, I meant using the issue in the todo as `TODO(<#issue-number>): <reason>`.

This was a part of my conditional LGTM. I will make it clearer in future reviews.